gallery/src/app/op-gallery/gallery.service.ts

```diff
@@ -12,6 +12,8 @@ export class GalleryService {
 
   private cachedResults: any[] = [];
 
+  sortType: string = 'default';
```

Comment:
rename it to `sortingType` and make it `private`

create two methods:

setSortingType(sortingType) {
 this.sortingType = sortingType
}

getSortingType() {
  return this.sortingType;
}